nss-c31-mixed-havoc/nutshell
# Bm styling friends

# Description

Adding bootstrap card styling to friends section. Also adding new jpg to be used when creating new user.

Applies to issue # (n/a)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout bm-styling-friends`
3. cd into /src/lib and run `grunt`
4. load `http://localhost:8080/` , log into the page with anyone's info, and verify that friends appear as cards with buttons headers, body, footers, and button styling to match the rest of the page.

# Checklist:
